HW03_waiy.py

```diff
@@ -0,0 +1,203 @@
+# TODO Do not edit this file directly! Instead, create a new file called
+# TODO a03_username.py and copy this code into it!
+
+#################################################################################
+# Author: Yin Kyay
+# Username: waiy
+#
+# Assignment: A03 Fully Functional Gitty Psychedelic Robotic Turtles
+# Purpose:
+# Google Doc Link: https://docs.google.com/document/d/1kZhHLMYkN9EUyj44LWp_RjF27UMaWgpc2mObpKcDIuA/edit
+#
+#################################################################################
+# Acknowledgements:
+
+#https://www.colorspire.com/rgb-color-wheel/
+#https://www.webucator.com/article/python-color-constants-module/
+
+#
+#
+#################################################################################
+
+
+
+
+
+
+import turtle
+
+
+def draw_square(t):
+
+
+   # the outer square
```

Comment:
docstring formatting is important (because it's how PyCharm and other IDEs know it's a docstring and not a regular comment, like this). 

The correct format is:


```
'''
 Docstring information, like what the function does
'''
```

Reply:
Fix this for every function.